include/lsst/afw/image/Exposure.h

```diff
@@ -391,6 +393,23 @@ class Exposure : public lsst::daf::base::Persistable, public lsst::daf::base::Ci
         return Exposure<ImageT, MaskT, VarianceT>(manager);
     }
 
+    /**
+     * Return an Exposure that is a small cutout of the original.
+     *
+     * @param center desired center of cutout (in RA and Dec)
+     * @param size width and height (in that order) of cutout in pixels
+     *
+     * @return An Exposure of the requested size centered on `center` to within
+     *     half a pixel in either dimension. Pixels past the edge of the original
+     *     exposure will be zero.
```

Comment:
I'd like to make sure I understand the desired behavior; I think you want to:
 - Ensure the given size is respected exactly.
 - Pad with zeros when the requested region is not wholly contained by the Exposure.
 - When a dimension of `size` is even (and hence the pixel-coordinate center of the returned subimage is a half-integer value in that dimension), the given `center` lands within 0.5 pixels of that half-integer value.
 - When a dimension of `size` is odd (and hence the pixel-coordinate center of the returned subimage is an integer value in that dimension), the given `center` lands within 0.5 pixels of that integer value.